jar-infer/tests/src/main/java/toys/Main.java

```diff
@@ -0,0 +1,67 @@
+package toys;
```

Comment:
We want some sort of unit tests that are run on NullAway's CI around these examples, and where we can check that we are getting the expected output. @msridhar Any ideas?

Maybe a http://wala.sourceforge.net/javadocs/trunk/com/ibm/wala/core/tests/util/WalaTestCase.html ?